src/NetworkTest/testConnectivity/index.ts

```diff
@@ -22,12 +23,32 @@ export type ConnectivityTestResults = {
   failedTests: FailureCase[],
 };
 
+/**
+ * Ensure that we're able to connect to the OpenTok API Server
+ */
+function connectToAPIServer(
+  OT: OpenTok,
+  { apiKey, sessionId, token }: SessionCredentials,
+): Promise<ConnectToAPIServerResults> {
+  const session = OT.initSession(apiKey, sessionId);
+  const fauxToken = 'T1==xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx==';
+  const fauxConnectionId = 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx';
+  return new Promise((resolve, reject) => {
+    const handleSuccess = () => resolve({ session, token });
+    OT.SessionInfo.get(session.sessionId, fauxToken, fauxConnectionId)
```

Comment:
`OT.SessionInfo.get()` is an unsupported API. It is subject to change. I recommend just making an XMLHttpRequest on Anvil.

Reply:
Actually, can we see whether the API server call failed based on the error returned in the `Session.connect()` completion handler? If so, then we should use that instead of writing our own call to the server.